doc/chapters/integration-guidelines/10-systemd-service.rst

```diff
@@ -0,0 +1,22 @@
+systemd.service
+===============
+
+As it is mentioned in :ref:`Getting Started <getting-started>` section the user API is implemented by
+the SoftwareContainer Agent as a D-bus service. The API is used to start, configure, and control
+containers. Thus in order to use SoftwareContainers the agent should be run first. To be able to start
+the agent and watch its process/system logs it would be wise decision to use systemd.service.
+
+systemd is an init system used in Linux distributions to bootstrap the user space. As of 2015, a large
+number of Linux distributions adopt systemd as their default init system. More information about this
+can be found `in freedesktop systemd docs <https://www.freedesktop.org/wiki/Software/systemd/>`_.
+
+To add the agent to init system, the integrator should prepare a configuration file whose name ends
+in ``.service``. This file encodes information about a process controlled and supervised by systemd.
+The ``.service`` files are generally kept in the ``/lib/systemd/system`` directory.
+
+Example
+-------
+The following sample will create a container where a write buffer will be
+created for all filesystems in that container
+
+.. literalinclude:: examples/softwarecontainer-agent.service
```

Comment:
I'm not sure we really want an example here, due to it creating a maintenance burden. How about just describing that there is special syntax for dbus services, and any runtime dependencies that the agent has?